core/lib/types/src/contract_verification/api.rs

```diff
@@ -147,6 +147,19 @@ pub struct VerificationIncomingRequest {
     pub is_system: bool,
     #[serde(default)]
     pub force_evmla: bool,
+    #[serde(flatten)]
+    pub evm_specific: VerificationEvmSettings,
+}
+
+/// Settings for EVM verification, used only if
+/// `SourceCodeData` is `SolSingleFile`.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct VerificationEvmSettings {
+    #[serde(default)]
+    pub evm_version: Option<String>,
+    #[serde(default)]
+    pub optimizer_runs: Option<u16>,
```

Comment:
I wonder whether it would be possible to put this data in the existing `optimizer_mode` column (it's already toolchain-specific, after all). Looks somewhat like a hack, so probably not worth it.